SearchHistory/tests.py

```diff
@@ -2,16 +2,9 @@
 from datetime import datetime
 import pytest
 
-PROFESSIONAL_ID = 101
-CLIENT_ID = 601
 DATE = datetime(2023, 4, 17, 10, 0, 0)
 
```

Comment:
Try to use relative times instead, like (now - 5 days). Decreases maintenance in tests' dates

Reply:
My mistake, the date is defaulted to be now

Reply:
Why did you delete the date?

Reply:
Because we need the default time and it is the current time that was at the time the object was created